controllers/indexController.js

```diff
@@ -0,0 +1,8 @@
+const getIndex = (req, res) => {
+    res.render("index", {
+        username: req.query.username ? req.query.username : "",
```

Comment:
```suggestion
        username: req.query.username ||  "",
```

it can be a bit shorter :)

Reply:
Done.